RELEASE_NOTES.md

```diff
@@ -27,240 +27,185 @@ will not be possible to downgrade from 3.0 to 2.6.x any more.
-  reordering. If you're not familiar with pixel pipe order and the
-  actual quality that could result from changing the default, do not
-  change the order.
-
-  The default order has been set to give the best result in the
-  majority of time and will never be on the wrong side.
-
-- The color zones module now display an histogram based on the ‘select
-  by’ channel and displays the selected range if the color picker is
-  in area mode.
-  It acts now like the tone curve (but horizontal),
-  it has two edit modes: edit by area is the former one, if not checked nodes
-  can be edited like in the tone curve, delete only works by right-click
-  when not in edit area mode.
-  A new process mode has been added: smooth is the former one, strong is new.
-  An interpolation method has been added that allows to select different types
-  of curves.
-  The color picker allows to select by area with <kbd>Ctrl+click</kbd>.
-  A new color picker has been added that creates a curve based on the area
-  selected from the image.
-  When pressing the shortcut 'pan&zoom while editing masks' the draw area
-  can now be zoomed. Double-click on the bottom bar resets the zoom.
-
-- New module Filmic RBG. This module is designed to replace the base
-  curve, shadows and highlights. The new filmic rgb module replaces
-  the old one introduced in 2.6.2 and should be easier to use and will
-  reduce color casts. The old filmic module is now deprecated and
-  cannot be used on new developments.
-
-- New module tone equalizer which brings an easy and safe way to
-  adjust the tones in different areas of the image. The module comes
-  with 9 controls ranging from blacks area to specular ones. It
-  possible to adjust each area with slides, curve or directly over the
-  image with a controllable picker.
+- The processing order of the pixel-pipe can now be changed by using
+  <kbd>Ctrl+Shift+drag</kbd> on the module headers to arrange them relative
+  to each other. It is VERY IMPORTANT to understand that this feature is NOT
+  for creating a more convenient GUI layout, but for changing the actual
+  processing of the image. DO NOT USE THIS FEATURE unless you understand the
+  reason behind the default ordering, and have a specific reason to change it.
+  The default order is still the correct order for most purposes.
+
+- The color zones module now shows a histogram based on the active ‘select
+  by’ channel, and will display the selected range within that channel if the
+  color picker is in 'select area' mode. By default, the spline adjustment is
+  now similar to the curve controls, allowing you to add and delete nodes, but
+  these nodes can still be moved with the old-style size control by selecting
+  the 'edit by area' checkbox. There is also a new 'strong' processing option,
+  and a new color picker which creates a curve based on selected image area.
+  
+- A new module 'filmic RGB' which, like the previous 'filmic', is designed
```

Comment:
May be use deprecated instead of previous.

Reply:
I'd actually like to leave this as is, it's explained as deprecated a few lines down, but some users might find "previous" a little easier to parse at first.